src/components/Match/MatchStory.jsx

```diff
@@ -540,6 +554,14 @@ const generateStory = (match) => {
   // Intro
   events.push(new IntroEvent(match));
 
+  // Prediction
+  match.players.forEach((player) => {
+    if (player.pred_vict === true) {
+      const order = player.player_slot < 5 ? player.player_slot - 89 : player.player_slot - 212;
```

Comment:
oh, all 10 have to be unique.  In that case why not just use player_slot directly?  They'll be unique, and I don't think there's a requirement for the range to be contiguous.